css/header.css

```diff
@@ -0,0 +1,719 @@
+    }
+
+    .header-nav-catalog-button{
+        display: none;
+    }
+
+    .header-nav-catalog-list{
+        display: flex;
+        align-items: center;
+        background-color: #DFDFDF;
+        text-decoration: none;
+        text-transform: uppercase;
+        padding: 0px;
+    }
+
+    .header-nav-catalog-list ul{
+        display: flex;
+        padding-left: 20px;
+    }
+
+    .header-nav-catalog-list li{
+        width: auto;
+        padding: 0px;
+        list-style-type: none;
+    }
+
+    .header-nav-catalog-list a{
+        font-weight: bold;
+        height: 50px;
+        color: black;
+        margin: 0px;
+        padding: 15px 10px;
+        white-space: nowrap;
+    }
+
+    .header-nav-catalog-list-list-button li{
+        padding: 10px 0;
+        text-transform: capitalize;
+        font-weight: normal;
+    }
+
+    .header-nav-catalog-list-list--scroll{
+        right: 10px!important;
+        left: 10px!important;
+    }
+
+    .header-nav-catalog-list-list{
+        display: none;
+        flex-direction: column;
+        position: absolute;
+        background-color: #ffffff;
+        margin-right: 0px;
+        top: 55px;
+        right: 0px;
+        left: 0px;
+        height: 200px;
+        flex-wrap: wrap;
+        padding: 30px 0;
+        box-shadow: 0px 0px 7px 0px #858585;
+        opacity: 0;
+        visibility: hidden;
+        transition: .5s;
+    }
+
+    .header-nav-catalog-list-list-button:hover .header-nav-catalog-list-list{
+        top: 50px;
+        height: 200px;
+        flex-wrap: wrap;
+        padding: 30px 0;
+        padding-left: 10px;
+        box-shadow: 0px 0px 7px 0px #858585;
+        opacity: 1;
+        visibility: visible;
+        transition: .5s;
+    }
+}
+
+@media screen and (max-width: 881px){
+    .header-wrapper{
+        margin: 30px 0;
+    }
+
+    .header-mid-logo{
+        width: 100%;
+    }
+
+    .header-mid-right{
+        width: 100%;
+        justify-content: space-between;
+    }
+
+    .header-nav{
+        margin-top: 20px;
+        justify-content: space-between;
+    }
+
+    .header-nav-menu-name{
+        width: 150px;
+    }
+
+    .header-nav-menu-list{
+        top: 0px;
+        left: 0px;
+        width: 200px;
+        position: fixed;
+        height: -webkit-fill-available;
+    }
+
+    .header-nav-catalog{
+        width: 150px;
+    }
+
+    .header-nav-catalog-button{
+        padding-left: 10px;
+        text-transform: uppercase;
+        color: #ffffff;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        flex-wrap: wrap;
+        box-sizing: border-box;
+    }
+
+    .header-nav-catalog-list{
+        margin: 0px;
+        background-color: #ffffff;
+        text-decoration: none;
+        text-transform: uppercase;
+        padding: 0px;
+        position: absolute;
+        width: 200px;
+        top: 0;
+        right: 0;
+        height: -webkit-fill-available;
```

Comment:
first of all it will work only in webkit-based browsers all other will ignore this rule.
second thing is: could we replace it with height 100%?
if it's not working it means that he cant calculate percentage of nothing. you should set some height for parent element and sometimes for parent parent element.